langchain/src/llms/openai-chat.ts

```diff
@@ -50,11 +50,9 @@ export interface OpenAIChatCallOptions extends OpenAICallOptions {
  * @augments AzureOpenAIChatInput
  */
 export class OpenAIChat
-  extends LLM
+  extends LLM<OpenAIChatCallOptions>
   implements OpenAIChatInput, AzureOpenAIInput
 {
-  declare CallOptions: OpenAIChatCallOptions;
```

Comment:
We dont need this anymore?

Reply:
Ah thanks, good catch

Reply:
Yeah, it's now declared with the type parameter on the superclass here: https://github.com/hwchase17/langchainjs/pull/2106/files/381bc22a2dd83824d3ff53574c78e3711e3d3c54#diff-0498c91d7202afcc70230e285dd4e46dfbd6c2525556c3b1cba7c3ce22e94671R115

Reply:
<img width="887" alt="Screenshot 2023-07-28 at 10 14 00 AM" src="https://github.com/hwchase17/langchainjs/assets/6952323/215ab687-601d-4c9c-9372-072a79e4c739">